src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppAppRunner.recipe

```diff
@@ -0,0 +1,285 @@
+{
+    "$schema": "./aws-deploy-recipe-schema.json",
+    "Id": "AspNetAppAppRunner",
+    "Version": "0.1.0",
+    "Name": "ASP.NET Core App to AWS App Runner",
+    "DeploymentType": "CdkProject",
+    "DeploymentBundle": "Container",
+    "CdkProjectTemplate": "../CdkTemplates/AspNetAppAppRunner",
+    "CdkProjectTemplateId": "netdeploy.AspNetAppAppRunner",
+    "Description": "Recommended for ASP.NET Core applications to be deployed in a fully management environment as a container image. If your project does not contain a Dockerfile, one will be generated for the project.",
+    "TargetService": "AWS App Runner",
+
+    "DisplayedResources": [
+        {
+            "LogicalId": "AppRunnerService",
+            "Description": "Application Endpoint"
+        }
+    ],
+
+    "RecipePriority": 105,
+    "RecommendationRules": [
+        {
+            "Tests": [
+                {
+                    "Type": "MSProjectSdkAttribute",
+                    "Condition": {
+                        "Value": "Microsoft.NET.Sdk.Web"
+                    }
+                },
+                {
+                    "Type": "MSProperty",
+                    "Condition": {
+                        "PropertyName": "TargetFramework",
+                        "AllowedValues": [ "netcoreapp2.1", "netcoreapp3.1", "net5.0", "net6.0" ]
+                    }
+                }
+            ]
+        },
+
+        {
+            "Tests": [
+                {
+                    "Type": "FileExists",
+                    "Condition": {
+                        "FileName": "Dockerfile"
+                    }
+                }
+            ],
+            "Effect": {
+                "Fail": {
+                    "PriorityAdjustment": -55,
```

Comment:
Is App Runner a service we want to promote? If so, maybe we shouldn't penalize for the lack of a dokerfile?

Reply:
This is not a promotion question. The same thought process as ECS is going on here. If they don't have Dockerfile then we assume it might not be their first priority to deploy to a container based service. 

We could consider reverse the logic and in the Beanstalk recipe if we detect there is not a Dockerfile then bump up the priority instead of putting the subtraction logic in all of the other docker based recipes. I would rather do that as a separate PR though.